pysph/tools/ipy_viewer.py

```diff
@@ -656,8 +676,26 @@ def _save_figure_handler(self, change):
         self._widgets.save_figure.value = ""
 
     def _delay_box_handler(self, change):
+
         self._widgets.play_button.interval = change['new']*1000
 
+    def _save_all_plots_handler(self, change):
+
+        if change['new']:
+            self._widgets.save_figure.disabled = True
+        elif not change['new']:
+            self._widgets.save_figure.disabled = False
+
+    def _save_plot(self):
+
+        abs_path = os.path.abspath(self.path)
+        if self._widgets.save_all_plots.value:
+            mkdir('/all_plots')
```

Comment:
This should really be `mkdir('all_plots')`!

Reply:
Corrected this in the new commit.